docs/developer-cheat-sheet.md

```diff
@@ -0,0 +1,61 @@
+# Developer Cheat Sheet
```

Comment:
Please make sure you agree with what's in here

Reply:
nice, an easy read